### PR TITLE
pre-commit lintr and styler

### PR DESCRIPTION
**Pull request by dhimmel**

### CODE REVIEW

**Comment by dhimmel** [comment]
I'm getting the following output when I run `Rscript 02.plot.R` so I might have broken something.

```
Some features are not enabled in this build of Arrow. Run `arrow_info()` for more information.

Attaching package: ‘arrow’

The following object is masked from ‘package:utils’:

    timestamp


Attaching package: ‘dplyr’

The following objects are masked from ‘package:stats’:

    filter, lag

The following objects are masked from ‘package:base’:

    intersect, setdiff, setequal, union


Attaching package: ‘cowplot’

The following object is masked from ‘package:patchwork’:

    align_plots

Warning messages:
1: In ggplot2::geom_segment(arrow = ggplot2::arrow(type = "open", length = grid::unit(arrow_length,  :
  All aesthetics have length 1, but the data has 436 rows.
ℹ Please consider using `annotate()` or provide this layer with data containing a single row.
2: No shared levels found between `names(values)` of the manual scale and the data's colour values. 
Warning messages:
1: In ggplot2::geom_segment(arrow = ggplot2::arrow(type = "open", length = grid::unit(arrow_length,  :
  All aesthetics have length 1, but the data has 436 rows.
ℹ Please consider using `annotate()` or provide this layer with data containing a single row.
2: No shared levels found between `names(values)` of the manual scale and the data's colour values. 
Warning messages:
1: In ggplot2::geom_segment(arrow = ggplot2::arrow(type = "open", length = grid::unit(arrow_length,  :
  All aesthetics have length 1, but the data has 436 rows.
ℹ Please consider using `annotate()` or provide this layer with data containing a single row.
2: No shared levels found between `names(values)` of the manual scale and the data's colour values. 
Error in transformation$transform(x) : 
  non-numeric argument to mathematical function
Calls: ggsave ... <Anonymous> -> transform_df -> lapply -> FUN -> transform
In addition: Warning message:
In ggplot2::geom_col(color = text_light, width = 360/ngr, fill = rose_pink) :
  All aesthetics have length 1, but the data has 32 rows.
ℹ Please consider using `annotate()` or provide this layer with data containing a single row.
Execution halted
`

**Comment by trangdata** [comment]
Could we make an exception for `ggplot2::` and `dplyr::` prefixes? Omitting these prefix (and using `library()` instead) makes the code a lot more readable, improving the overall clarity, especially when we use these functions so frequently. For example, I think spelling out `ggplot::aes()` in every plot is 🤯

File: r/02.plot.R [hunk 1]
```diff
@@ -1,10 +1,6 @@
-library(arrow)
-library(dplyr)
 library(ggplot2)
-library(yaml)
-library(patchwork)
-library(cowplot)
-library(svglite)
+# so that we don't have to do ggplot2::element_line a zillion times
```
Comment (dhimmel) [inline r/02.plot.R:2]:
```suggestion
# so that we don't have to do patchwork::area a zillion times
```

**Comment by trangdata** [comment]
Hmm I removed the action cache right before this last trigger but the runs still failed... Any idea @dhimmel ?

**Pull request merged**